packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -152,7 +152,11 @@ class BorderTween extends Tween<Border> {
 
   /// Returns the value this variable has at the given animation clock value.
   @override
-  Border lerp(double t) => Border.lerp(begin, end, t)!;
+  Border? lerp(double t) {
+    if (begin == null || end == null)
+      return t < 0.5 ? begin : end;
```

Comment:
@HansMuller Hey, Would like to ask why this is handled here instead of as follow:
```dart
Border? lerp(double t) => Border.lerp(begin, end, t);
```

Reply:
Good point, I think that would be simpler and clearer. Can you create a PR with the change?

Reply:
Ok

Reply:
update pr #79919